specification/communication/data-plane/Sms/preview/2024-01-14-preview/communicationservicessms.json

```diff
@@ -0,0 +1,436 @@
+          }
+        },
+        "sendOptions": {
+          "$ref": "#/definitions/MmsSendOptions"
+        }
+      }
+    },
+    "MmsRecipient": {
+      "description": "Represents properties for a single recipient.",
+      "required": [
+        "to"
+      ],
+      "type": "object",
+      "properties": {
+        "to": {
+          "description": "The recipient's phone number in E.164 format.",
+          "minLength": 1,
+          "type": "string"
+        },
+        "repeatabilityRequestId": {
+          "description": "If specified, the client directs that the request is repeatable; that is, the client can make the request multiple times with the same Repeatability-Request-ID and get back an appropriate response without the server executing the request multiple times. The value of the Repeatability-Request-ID is an opaque string representing a client-generated, 36-character hexadecimal case-insensitive encoding of a UUID (GUID), identifier for the request.",
+          "type": "string"
+        },
+        "repeatabilityFirstSent": {
+          "description": "MUST be sent by clients to specify that a request is repeatable. Repeatability-First-Sent is used to specify the date and time at which the request was first created.eg- Tue, 26 Mar 2019 16:06:51 GMT.",
+          "type": "string"
+        }
+      }
+    },
+    "MmsSendOptions": {
+      "description": "Optional configuration for sending MMS messages.",
+      "required": [
+        "enableDeliveryReport"
+      ],
+      "type": "object",
+      "properties": {
+        "enableDeliveryReport": {
+          "description": "Enable this flag to receive a delivery report for this message on the Azure Resource EventGrid.",
+          "type": "boolean"
+        },
+        "tag": {
+          "description": "Use this field to provide metadata that will then be sent back in the corresponding Delivery Report.",
+          "type": "string"
+        }
+      }
+    },
+    "MmsSendRequestAttachment": {
+      "description": "Represents the properties of a send request attachment.",
+      "required": [
+        "contentInBase64",
+        "contentType"
+      ],
+      "type": "object",
+      "properties": {
+        "contentType": {
+          "description": "MIME type of attachment.",
+          "enum": [
+            "image/png",
+            "image/jpeg",
+            "image/gif",
+            "image/bmp",
+            "audio/wav",
+            "audio/x-wav",
+            "audio/ac3",
+            "audio/amr",
+            "video/mp4",
+            "video/x-msvideo",
+            "text/plain"
+          ],
+          "type": "string",
+          "example": "image/png",
+          "x-ms-enum": {
+            "name": "MmsContentType",
+            "modelAsString": true
+          }
+        },
+        "contentInBase64": {
+          "description": "Content of the attachment encoded in base 64.",
+          "minLength": 1,
+          "type": "string",
```

Comment:
I think we should add `"format": "byte",` which we also use for Email
https://github.com/Azure/azure-rest-api-specs/blob/69f5096c9558b8f4ecb2b0a0c516a10fce58e03e/specification/communication/data-plane/Email/stable/2023-03-31/CommunicationServicesEmail.json#L308

Reply:
ok, added